kbase-extension/scss/partials/_buttons.scss

```diff
@@ -22,3 +22,24 @@
   /* don't add shadows here */
   box-shadow: none;
 }
+
+
+.btn-action {
```

Comment:
Good idea!

Reply:
Let's standardise and have kbase css classes start with the prefix `kb-` -- it will enable us to differentiate (to some extent) between what we've written and what is a bootstrap / select2 / datatables / etc. override.